{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Import Required Libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n and I love you",
    "from bs4 import BeautifulSoup"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Defining gene id and attributes to be extracted"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "arr=[\"GSE63312\",\"GSE78224,\",\"GSE74018\", \"GSE50734\", \"GSE114644\", \"GSE60477\", \"GSE53599\",\n",
    "\"GSE80582\", \"GSE109493\", \"GSE35200\"]\n",
    "a=[\"Status\",\"Title\",\"Distribution\",\"Organism\",\"Experiment type\",\"Summary\",\"Overall design\",\"Contributor(s)\",\"Citation(s)\"]\n",
    "data=[]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Function to annotate the summary attribute by BERN API"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def query_raw(text, url=\"https://bern.korea.ac.kr/plain\"):\n",
    "    return requests.post(url, data={'sample_text': text}).json()\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Writing the scraping code and iterating for all gene id's"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in arr:\n",
    "    res=requests.get('https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc='+str(i))\n",
    "    soup=BeautifulSoup(res.text,\"html.parser\")\n",
    "    d={}\n",
    "    for j in a:\n",
    "        body=soup.find(text=j)\n",
    "        if(body!=None):\n",
    "            d[str(j)]=body.findNext('td').text\n",
    "    data.append(d)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Appending annotations to the data extracted"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in data:\n",
    "    if \"Summary\" in i.keys():\n",
    "        i['annotate']=query_raw(i[\"Summary\"])\n",
    "        "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Printing the Data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Status': 'Public on Nov 14, 2014', 'Title': 'Leukemia stem cell-enriched population expresses self-renewal gene-expression signature [RNA-Seq]', 'Organism': 'Mus musculus', 'Experiment type': 'Expression profiling by high throughput sequencing', 'Summary': 'Mutant RAS oncoproteins activate signaling molecules that drive oncogenesis in multiple human tumors including acute myelogenous leukemia (AML). However, the specific function of these pathways in AML is unclear. To elucidate the downstream functions of activated NRAS in AML, we employed a murine model of AML harboring Mll-AF9 and NRASG12V. We found that NRASG12V enforced leukemia self-renewal gene expression signatures and was required to maintain an MLL-AF9 and MYB-dependent gene expression program. In a multiplexed analysis of RAS-dependent signaling intermediates, the leukemia stem cell compartment was preferentially sensitive to RAS withdrawal. Use of RAS-pathway inhibitors showed that NRASG12V maintained leukemia self-renewal through mTOR and MEK pathway activation, implicating these pathways as potential targets for cancer stem cell-specific therapies.', 'Overall design': 'Primary leukemia cells harvested from spleens were sorted into immunophenotypic subpopulations (Mac-1High, Mac-1LowKitâ\\x80\\x93Sca-1â\\x80\\x93, Mac-1LowKit+Sca-1â\\x80\\x93, and Mac-1LowKit+Sca-1+). RNA was extracted from this subpopulations of cells and submitted for RNA sequencing.', 'Contributor(s)': 'Sachs Z, LaRue RS, Nguyen HT, Sachs K, Hassan NA, Diaz-Flores E, Rathe SK, Sarver AL, Bendall SC, Ha NA, Diers MD, Nolan GP, Shannon KM, Largaespada DA', 'Citation(s)': '25316678', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': '94c48ca615247e0ca7183da6060d22f31babc695763c448e78967185-Thread-14989849', 'text': 'Mutant RAS oncoproteins activate signaling molecules that drive oncogenesis in multiple human tumors including acute myelogenous leukemia (AML). However, the specific function of these pathways in AML is unclear. To elucidate the downstream functions of activated NRAS in AML, we employed a murine model of AML harboring Mll-AF9 and NRASG12V. We found that NRASG12V enforced leukemia self-renewal gene expression signatures and was required to maintain an MLL-AF9 and MYB-dependent gene expression program. In a multiplexed analysis of RAS-dependent signaling intermediates, the leukemia stem cell compartment was preferentially sensitive to RAS withdrawal. Use of RAS-pathway inhibitors showed that NRASG12V maintained leukemia self-renewal through mTOR and MEK pathway activation, implicating these pathways as potential targets for cancer stem cell-specific therapies.', 'denotations': [{'id': ['CUI-less'], 'span': {'begin': 7, 'end': 23}, 'obj': 'gene'}, {'id': ['BERN:256225101'], 'span': {'begin': 94, 'end': 100}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 111, 'end': 137}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 139, 'end': 142}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 197, 'end': 200}, 'obj': 'disease'}, {'id': ['MIM:164790', 'HGNC:7989', 'Ensembl:ENSG00000213281', 'BERN:324183902'], 'span': {'begin': 264, 'end': 268}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 272, 'end': 275}, 'obj': 'disease'}, {'id': ['NCBI:txid10090'], 'span': {'begin': 291, 'end': 297}, 'obj': 'species'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 307, 'end': 310}, 'obj': 'disease'}, {'id': ['MIM:159555', 'HGNC:7132', 'Ensembl:ENSG00000118058', 'BERN:324047002'], 'span': {'begin': 321, 'end': 328}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 333, 'end': 341}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 357, 'end': 365}, 'obj': 'gene'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 375, 'end': 383}, 'obj': 'disease'}, {'id': ['MIM:159555', 'HGNC:7132', 'Ensembl:ENSG00000118058', 'BERN:324047002'], 'span': {'begin': 456, 'end': 459}, 'obj': 'gene'}, {'id': ['MIM:159558', 'HGNC:7136', 'Ensembl:ENSG00000171843', 'BERN:324047502'], 'span': {'begin': 460, 'end': 463}, 'obj': 'gene'}, {'id': ['MIM:189990', 'HGNC:7545', 'Ensembl:ENSG00000118513', 'BERN:324141502'], 'span': {'begin': 468, 'end': 471}, 'obj': 'gene'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 579, 'end': 587}, 'obj': 'disease'}, {'id': ['BERN:326404802'], 'span': {'begin': 665, 'end': 668}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 700, 'end': 708}, 'obj': 'gene'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 720, 'end': 728}, 'obj': 'disease'}, {'id': ['MIM:601231', 'HGNC:3942', 'Ensembl:ENSG00000198793', 'BERN:323251002'], 'span': {'begin': 750, 'end': 754}, 'obj': 'gene'}, {'id': ['MIM:603014', 'HGNC:6847', 'Ensembl:ENSG00000076984', 'BERN:324467602'], 'span': {'begin': 759, 'end': 762}, 'obj': 'gene'}, {'id': ['MESH:D002277', 'BERN:255268301'], 'span': {'begin': 835, 'end': 841}, 'obj': 'disease'}], 'timestamp': 'Sun Jul 18 17:34:46 +0000 2021', 'logits': {'disease': [[{'start': 94, 'end': 100, 'id': 'BERN:256225101'}, 0.9999997615814209], [{'start': 111, 'end': 137, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998211860657], [{'start': 139, 'end': 142, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 197, 'end': 200, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 272, 'end': 275, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 307, 'end': 310, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 375, 'end': 383, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999998807907104], [{'start': 579, 'end': 587, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999998807907104], [{'start': 720, 'end': 728, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999998807907104], [{'start': 835, 'end': 841, 'id': 'MESH:D002277\\tBERN:255268301'}, 0.9999998807907104]], 'gene': [[{'start': 7, 'end': 23, 'id': 'CUI-less'}, 0.9999598860740662], [{'start': 264, 'end': 268, 'id': 'MIM:164790\\tHGNC:7989\\tEnsembl:ENSG00000213281\\tBERN:324183902'}, 0.9999972581863403], [{'start': 321, 'end': 328, 'id': 'MIM:159555\\tHGNC:7132\\tEnsembl:ENSG00000118058\\tBERN:324047002'}, 0.999997079372406], [{'start': 333, 'end': 341, 'id': 'CUI-less'}, 0.9999972581863403], [{'start': 357, 'end': 365, 'id': 'CUI-less'}, 0.9999972581863403], [{'start': 456, 'end': 459, 'id': 'MIM:159555\\tHGNC:7132\\tEnsembl:ENSG00000118058\\tBERN:324047002'}, 0.9999972581863403], [{'start': 460, 'end': 463, 'id': 'MIM:159558\\tHGNC:7136\\tEnsembl:ENSG00000171843\\tBERN:324047502'}, 0.8917233347892761], [{'start': 468, 'end': 471, 'id': 'MIM:189990\\tHGNC:7545\\tEnsembl:ENSG00000118513\\tBERN:324141502'}, 0.9999974966049194], [{'start': 665, 'end': 668, 'id': 'BERN:326404802'}, 0.9999972581863403], [{'start': 700, 'end': 708, 'id': 'CUI-less'}, 0.9999877214431763], [{'start': 750, 'end': 754, 'id': 'MIM:601231\\tHGNC:3942\\tEnsembl:ENSG00000198793\\tBERN:323251002'}, 0.9999973773956299], [{'start': 759, 'end': 762, 'id': 'MIM:603014\\tHGNC:6847\\tEnsembl:ENSG00000076984\\tBERN:324467602'}, 0.9999972581863403]], 'drug': [], 'species': [[{'start': 88, 'end': 93}, 0.9999998807907104], [{'start': 291, 'end': 297, 'id': 'NCBI:txid10090'}, 0.9999997615814209]]}}}\n",
      "{}\n",
      "{'Status': 'Public on Oct 18, 2016', 'Title': 'Acute myeloid leukemia requires Hhex to enable PRC2-mediated epigenetic repression of Cdkn2a (RNA-Seq)', 'Organism': 'Mus musculus', 'Experiment type': 'Expression profiling by high throughput sequencing', 'Summary': 'Unlike clustered HOX genes, the role of non-clustered homeobox gene family members in hematopoiesis and leukemogenesis has not been extensively studied. Here we find that the Hematopoietically-expressed Homeobox gene, Hhex, is overexpressed in acute myeloid leukemia (AML) and is essential for the initiation and propagation of MLL-ENL induced AML, but dispensable for normal myelopoiesis, indicating a specific requirement for Hhex for leukemic growth. Loss of Hhex leads to upregulated expression of the tumor suppressors p16INK4a and p19ARF, which are required for growth arrest and myeloid differentiation following Hhex deletion. Mechanistically, we show that Hhex binds to the Cdkn2a locus and is required for H3K27Me3-mediated repression of this locus. Thus, Hhex is a novel therapeutic target that is specifically required for AML stem cells to repress tumor suppressor pathways and enable continued self-renewal.', 'Overall design': 'Transcriptional profiling of MLL-ENL Hhex-/fl and MLL-ENL Hhex+/fl leukemia cells using RNA sequencing', 'Contributor(s)': 'Shields BJ, Shi W, Garnham AL, Smyth GK, McCormack MP', 'Citation(s)': '26728554', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': '116002e48c9dabb6facd177b652218126e875dae5f146c584b3719b0-Thread-14989862', 'text': 'Unlike clustered HOX genes, the role of non-clustered homeobox gene family members in hematopoiesis and leukemogenesis has not been extensively studied. Here we find that the Hematopoietically-expressed Homeobox gene, Hhex, is overexpressed in acute myeloid leukemia (AML) and is essential for the initiation and propagation of MLL-ENL induced AML, but dispensable for normal myelopoiesis, indicating a specific requirement for Hhex for leukemic growth. Loss of Hhex leads to upregulated expression of the tumor suppressors p16INK4a and p19ARF, which are required for growth arrest and myeloid differentiation following Hhex deletion. Mechanistically, we show that Hhex binds to the Cdkn2a locus and is required for H3K27Me3-mediated repression of this locus. Thus, Hhex is a novel therapeutic target that is specifically required for AML stem cells to repress tumor suppressor pathways and enable continued self-renewal.', 'denotations': [{'id': ['CUI-less'], 'span': {'begin': 17, 'end': 26}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 54, 'end': 74}, 'obj': 'gene'}, {'id': ['BERN:257919501'], 'span': {'begin': 104, 'end': 118}, 'obj': 'disease'}, {'id': ['CUI-less'], 'span': {'begin': 175, 'end': 202}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 203, 'end': 216}, 'obj': 'gene'}, {'id': ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], 'span': {'begin': 218, 'end': 222}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 244, 'end': 266}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 268, 'end': 271}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 344, 'end': 347}, 'obj': 'disease'}, {'id': ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], 'span': {'begin': 428, 'end': 432}, 'obj': 'gene'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 437, 'end': 445}, 'obj': 'disease'}, {'id': ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], 'span': {'begin': 462, 'end': 466}, 'obj': 'gene'}, {'id': ['BERN:256225101'], 'span': {'begin': 506, 'end': 511}, 'obj': 'disease'}, {'id': ['MIM:600160', 'HGNC:1787', 'Ensembl:ENSG00000147889', 'BERN:321173202'], 'span': {'begin': 524, 'end': 532}, 'obj': 'gene'}, {'id': ['MIM:600160', 'HGNC:1787', 'Ensembl:ENSG00000147889', 'BERN:321173202'], 'span': {'begin': 537, 'end': 543}, 'obj': 'gene'}, {'id': ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], 'span': {'begin': 620, 'end': 624}, 'obj': 'gene'}, {'id': ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], 'span': {'begin': 665, 'end': 669}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 683, 'end': 695}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 716, 'end': 724}, 'obj': 'gene'}, {'id': ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], 'span': {'begin': 766, 'end': 770}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 835, 'end': 838}, 'obj': 'disease'}, {'id': ['BERN:256225101'], 'span': {'begin': 861, 'end': 866}, 'obj': 'disease'}], 'timestamp': 'Sun Jul 18 17:34:50 +0000 2021', 'logits': {'disease': [[{'start': 104, 'end': 118, 'id': 'BERN:257919501'}, 0.9999825954437256], [{'start': 244, 'end': 266, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999999403953552], [{'start': 268, 'end': 271, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 344, 'end': 347, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999995231628418], [{'start': 437, 'end': 445, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999964237213135], [{'start': 506, 'end': 511, 'id': 'BERN:256225101'}, 0.9999998807907104], [{'start': 835, 'end': 838, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 861, 'end': 866, 'id': 'BERN:256225101'}, 0.9999998807907104]], 'gene': [[{'start': 17, 'end': 26, 'id': 'CUI-less'}, 0.9999959468841553], [{'start': 54, 'end': 74, 'id': 'CUI-less'}, 0.9999960064888], [{'start': 175, 'end': 202, 'id': 'CUI-less'}, 0.9292442798614502], [{'start': 203, 'end': 216, 'id': 'CUI-less'}, 0.9999106526374817], [{'start': 218, 'end': 222, 'id': 'MIM:604420\\tHGNC:4901\\tEnsembl:ENSG00000152804\\tBERN:323596902'}, 0.9999972581863403], [{'start': 428, 'end': 432, 'id': 'MIM:604420\\tHGNC:4901\\tEnsembl:ENSG00000152804\\tBERN:323596902'}, 0.9999970197677612], [{'start': 462, 'end': 466, 'id': 'MIM:604420\\tHGNC:4901\\tEnsembl:ENSG00000152804\\tBERN:323596902'}, 0.9999973773956299], [{'start': 524, 'end': 532, 'id': 'MIM:600160\\tHGNC:1787\\tEnsembl:ENSG00000147889\\tBERN:321173202'}, 0.9999971389770508], [{'start': 537, 'end': 543, 'id': 'MIM:600160\\tHGNC:1787\\tEnsembl:ENSG00000147889\\tBERN:321173202'}, 0.9999972581863403], [{'start': 620, 'end': 624, 'id': 'MIM:604420\\tHGNC:4901\\tEnsembl:ENSG00000152804\\tBERN:323596902'}, 0.9999973773956299], [{'start': 665, 'end': 669, 'id': 'MIM:604420\\tHGNC:4901\\tEnsembl:ENSG00000152804\\tBERN:323596902'}, 0.9999972581863403], [{'start': 683, 'end': 695, 'id': 'CUI-less'}, 0.999992847442627], [{'start': 716, 'end': 724, 'id': 'CUI-less'}, 0.9999972581863403], [{'start': 766, 'end': 770, 'id': 'MIM:604420\\tHGNC:4901\\tEnsembl:ENSG00000152804\\tBERN:323596902'}, 0.9999969005584717]], 'drug': [], 'species': []}}}\n",
      "{'Status': 'Public on Nov 04, 2013', 'Title': 'TEL-AML1 (ETV6-RUNX1) in B-cells and leukemia (part 5)', 'Organism': 'Homo sapiens', 'Experiment type': 'Expression profiling by array', 'Summary': 'We identified directly and indirectly regulated target genes utilizing an inducible TEL-AML1 system derived from the murine pro B-cell line BA/F3 and a monoclonal antibody directed against TEL-AML1. By integration of promoter binding identified with ChIP-on-chip, gene expression and protein output through microarray technology and stable labelling of amino acids in cell culture (SILAC), we identified directly and indirectly regulated targets of the TEL-AML1 fusion protein.', 'Overall design': 'Bound promoter regions of immunoprecipitated TEL-AML1 associated genes were first compared to input material and enrichment was calculated. The same was performed for empty vector control cell lines, also treated with the induction reagent mifepriston. Enriched promoter regions were then compared of the both sets. 2 independent replicates each (4 arrays) were performed.', 'Contributor(s)': 'Linka Y, Ginzel S, KrÃ¼ger M, Novosel A, Gombert M, Kremmer E, Harbott J, Thiele R, Borkhardt A, Landgraf P', 'Citation(s)': '24121163, 26484077', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': 'df8b2575f47eb2f8ac941e013b5ac972ab67fabdfd51f961ca3a4973-Thread-14989875', 'text': 'We identified directly and indirectly regulated target genes utilizing an inducible TEL-AML1 system derived from the murine pro B-cell line BA/F3 and a monoclonal antibody directed against TEL-AML1. By integration of promoter binding identified with ChIP-on-chip, gene expression and protein output through microarray technology and stable labelling of amino acids in cell culture (SILAC), we identified directly and indirectly regulated targets of the TEL-AML1 fusion protein.', 'denotations': [{'id': ['MIM:600618', 'HGNC:3495', 'Ensembl:ENSG00000139083', 'BERN:323103602'], 'span': {'begin': 84, 'end': 87}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 88, 'end': 92}, 'obj': 'gene'}, {'id': ['NCBI:txid10090'], 'span': {'begin': 117, 'end': 123}, 'obj': 'species'}, {'id': ['MIM:600618', 'HGNC:3495', 'Ensembl:ENSG00000139083', 'BERN:323103602'], 'span': {'begin': 189, 'end': 192}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 193, 'end': 197}, 'obj': 'gene'}, {'id': ['CHEBI:33704', 'BERN:282227403'], 'span': {'begin': 353, 'end': 364}, 'obj': 'drug'}, {'id': ['MIM:600618', 'HGNC:3495', 'Ensembl:ENSG00000139083', 'BERN:323103602'], 'span': {'begin': 453, 'end': 456}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 457, 'end': 461}, 'obj': 'gene'}], 'timestamp': 'Sun Jul 18 17:34:53 +0000 2021', 'logits': {'disease': [], 'gene': [[{'start': 84, 'end': 87, 'id': 'MIM:600618\\tHGNC:3495\\tEnsembl:ENSG00000139083\\tBERN:323103602'}, 0.9999958276748657], [{'start': 88, 'end': 92, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999957084655762], [{'start': 189, 'end': 192, 'id': 'MIM:600618\\tHGNC:3495\\tEnsembl:ENSG00000139083\\tBERN:323103602'}, 0.9999972581863403], [{'start': 193, 'end': 197, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999971389770508], [{'start': 453, 'end': 456, 'id': 'MIM:600618\\tHGNC:3495\\tEnsembl:ENSG00000139083\\tBERN:323103602'}, 0.9999973773956299], [{'start': 457, 'end': 461, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999973773956299]], 'drug': [[{'start': 353, 'end': 364, 'id': 'CHEBI:33704\\tBERN:282227403'}, 0.9999977350234985]], 'species': [[{'start': 117, 'end': 123, 'id': 'NCBI:txid10090'}, 0.9999998807907104]]}}}\n",
      "{'Status': 'Public on Dec 14, 2018', 'Title': 'Different roles of E proteins in t(8;21) leukemia: E2-2 compromises the function of AETFC and negatively regulates leukemogenesis', 'Organism': 'Homo sapiens', 'Experiment type': 'Expression profiling by high throughput sequencingGenome binding/occupancy profiling by high throughput sequencing', 'Summary': 'The AML1-ETO fusion protein, generated by the t(8;21) chromosomal translocation, is causally involved in nearly 20% of acute myeloid leukemia (AML) cases. In leukemic cells, AML1-ETO resides in and functions through a stable protein complex, AETFC, that contains multiple transcription (co)factors. Among these AETFC components, HEB and E2A, two members of the ubiquitously expressed E proteins, directly interact with AML1-ETO, confer new DNA-binding capacity to AETFC, and are essential for leukemogenesis. However, the third E protein, E2-2, is specifically silenced in AML1-ETOâ\\x80\\x93expressing leukemic cells, suggesting E2-2 as a negative factor of leukemogenesis. Indeed, ectopic expression of E2-2 selectively inhibits the growth of AML1-ETOâ\\x80\\x93expressing leukemic cells, and this inhibition requires the bHLH DNA-binding domain. RNA-seq and ChIP-seq analyses reveal that, despite overlapping to some extent, the three E proteins differentially regulate many target genes, and, in particular, that E2-2 both redistributes AETFC to, and activates, some genes associated with dendritic cell differentiation. In AML patients, the expression of E2-2 is relatively lower in the t(8;21) subtype, and an E2-2 target gene, THPO, is identified as a potential predictor of relapse. In a mouse model of human t(8;21) leukemia, E2-2 suppression accelerates leukemogenesis. Taken together, these results reveal that, in contrast to HEB and E2A, which facilitate AML1-ETOâ\\x80\\x93mediated leukemogenesis, E2-2 compromises the function of AETFC and acts as a negative regulator of leukemogenesis. The three E proteins thus define a heterogeneity of AETFC, which improves our understanding of the precise mechanism of leukemogenesis and assists development of diagnostic/therapeutic strategies.', 'Overall design': 'The three E proteins (E2-2, E2A and HEB) were tagged with HA and transduced into Kasumi-1 cells and the ChIP-seq was performed with an anti-HA antibody. These cells, as well as the control cells transduced with the MIGR1 vector, were also subjected to RNA-seq analysis.', 'Contributor(s)': 'Liu N, Rong B, Xu C, Lan F, Wang L, Sun X', 'Citation(s)': '30593567', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': '05006e980c7bd5edb5302d6c88e676e195b7fe875b4b31fc16592429-Thread-14989888', 'text': 'The AML1-ETO fusion protein, generated by the t(8;21) chromosomal translocation, is causally involved in nearly 20% of acute myeloid leukemia (AML) cases. In leukemic cells, AML1-ETO resides in and functions through a stable protein complex, AETFC, that contains multiple transcription (co)factors. Among these AETFC components, HEB and E2A, two members of the ubiquitously expressed E proteins, directly interact with AML1-ETO, confer new DNA-binding capacity to AETFC, and are essential for leukemogenesis. However, the third E protein, E2-2, is specifically silenced in AML1-ETO   expressing leukemic cells, suggesting E2-2 as a negative factor of leukemogenesis. Indeed, ectopic expression of E2-2 selectively inhibits the growth of AML1-ETO   expressing leukemic cells, and this inhibition requires the bHLH DNA-binding domain. RNA-seq and ChIP-seq analyses reveal that, despite overlapping to some extent, the three E proteins differentially regulate many target genes, and, in particular, that E2-2 both redistributes AETFC to, and activates, some genes associated with dendritic cell differentiation. In AML patients, the expression of E2-2 is relatively lower in the t(8;21) subtype, and an E2-2 target gene, THPO, is identified as a potential predictor of relapse. In a mouse model of human t(8;21) leukemia, E2-2 suppression accelerates leukemogenesis. Taken together, these results reveal that, in contrast to HEB and E2A, which facilitate AML1-ETO   mediated leukemogenesis, E2-2 compromises the function of AETFC and acts as a negative regulator of leukemogenesis. The three E proteins thus define a heterogeneity of AETFC, which improves our understanding of the precise mechanism of leukemogenesis and assists development of diagnostic/therapeutic strategies.', 'denotations': [{'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 4, 'end': 8}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 9, 'end': 12}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 119, 'end': 141}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 143, 'end': 146}, 'obj': 'disease'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 158, 'end': 166}, 'obj': 'disease'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 174, 'end': 178}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 179, 'end': 182}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 242, 'end': 247}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 311, 'end': 316}, 'obj': 'gene'}, {'id': ['MIM:600480', 'HGNC:11623', 'Ensembl:ENSG00000140262', 'BERN:324936702'], 'span': {'begin': 329, 'end': 332}, 'obj': 'gene'}, {'id': ['MIM:147141', 'HGNC:11633', 'Ensembl:ENSG00000071564', 'BERN:324924802'], 'span': {'begin': 337, 'end': 340}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 384, 'end': 394}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 419, 'end': 423}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 424, 'end': 427}, 'obj': 'gene'}, {'id': ['BERN:257919501'], 'span': {'begin': 493, 'end': 507}, 'obj': 'disease'}, {'id': ['CUI-less'], 'span': {'begin': 528, 'end': 537}, 'obj': 'gene'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 539, 'end': 543}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 573, 'end': 577}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 578, 'end': 581}, 'obj': 'gene'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 595, 'end': 603}, 'obj': 'disease'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 622, 'end': 626}, 'obj': 'gene'}, {'id': ['BERN:257919501'], 'span': {'begin': 651, 'end': 665}, 'obj': 'disease'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 697, 'end': 701}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 737, 'end': 741}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 742, 'end': 745}, 'obj': 'gene'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 759, 'end': 767}, 'obj': 'disease'}, {'id': ['CUI-less'], 'span': {'begin': 922, 'end': 932}, 'obj': 'gene'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 1001, 'end': 1005}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 1025, 'end': 1030}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 1112, 'end': 1115}, 'obj': 'disease'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 1144, 'end': 1148}, 'obj': 'gene'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 1200, 'end': 1204}, 'obj': 'gene'}, {'id': ['MESH:C026804', 'BERN:317251403'], 'span': {'begin': 1218, 'end': 1222}, 'obj': 'drug'}, {'id': ['NCBI:txid10090'], 'span': {'begin': 1280, 'end': 1285}, 'obj': 'species'}, {'id': ['MESH:D007938', 'BERN:106966401'], 'span': {'begin': 1301, 'end': 1317}, 'obj': 'disease'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 1319, 'end': 1323}, 'obj': 'gene'}, {'id': ['BERN:257919501'], 'span': {'begin': 1348, 'end': 1362}, 'obj': 'disease'}, {'id': ['MIM:600480', 'HGNC:11623', 'Ensembl:ENSG00000140262', 'BERN:324936702'], 'span': {'begin': 1422, 'end': 1425}, 'obj': 'gene'}, {'id': ['MIM:147141', 'HGNC:11633', 'Ensembl:ENSG00000071564', 'BERN:324924802'], 'span': {'begin': 1430, 'end': 1433}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 1452, 'end': 1456}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 1457, 'end': 1460}, 'obj': 'gene'}, {'id': ['BERN:257919501'], 'span': {'begin': 1472, 'end': 1486}, 'obj': 'disease'}, {'id': ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], 'span': {'begin': 1488, 'end': 1492}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 1521, 'end': 1526}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 1589, 'end': 1599}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 1631, 'end': 1636}, 'obj': 'gene'}], 'timestamp': 'Sun Jul 18 17:35:02 +0000 2021', 'logits': {'disease': [[{'start': 119, 'end': 141, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999999403953552], [{'start': 143, 'end': 146, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999997615814209], [{'start': 158, 'end': 166, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999995231628418], [{'start': 493, 'end': 507, 'id': 'BERN:257919501'}, 0.9999949932098389], [{'start': 595, 'end': 603, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999997615814209], [{'start': 651, 'end': 665, 'id': 'BERN:257919501'}, 0.9992891550064087], [{'start': 759, 'end': 767, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999996423721313], [{'start': 1112, 'end': 1115, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 1301, 'end': 1317, 'id': 'MESH:D007938\\tBERN:106966401'}, 0.9999945759773254], [{'start': 1348, 'end': 1362, 'id': 'BERN:257919501'}, 0.9999994039535522], [{'start': 1472, 'end': 1486, 'id': 'BERN:257919501'}, 0.9999984502792358]], 'gene': [[{'start': 4, 'end': 8, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999972581863403], [{'start': 9, 'end': 12, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999973773956299], [{'start': 174, 'end': 178, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.999996542930603], [{'start': 179, 'end': 182, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999972581863403], [{'start': 242, 'end': 247, 'id': 'CUI-less'}, 0.9999972581863403], [{'start': 311, 'end': 316, 'id': 'CUI-less'}, 0.9999973773956299], [{'start': 329, 'end': 332, 'id': 'MIM:600480\\tHGNC:11623\\tEnsembl:ENSG00000140262\\tBERN:324936702'}, 0.9999972581863403], [{'start': 337, 'end': 340, 'id': 'MIM:147141\\tHGNC:11633\\tEnsembl:ENSG00000071564\\tBERN:324924802'}, 0.9999971389770508], [{'start': 384, 'end': 394, 'id': 'CUI-less'}, 0.9999955892562866], [{'start': 419, 'end': 423, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999973773956299], [{'start': 424, 'end': 427, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999973773956299], [{'start': 528, 'end': 537, 'id': 'CUI-less'}, 0.9998805522918701], [{'start': 539, 'end': 543, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999970197677612], [{'start': 573, 'end': 577, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9992844462394714], [{'start': 578, 'end': 581, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999973773956299], [{'start': 622, 'end': 626, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999970197677612], [{'start': 697, 'end': 701, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999970197677612], [{'start': 737, 'end': 741, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9998819828033447], [{'start': 742, 'end': 745, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999972581863403], [{'start': 922, 'end': 932, 'id': 'CUI-less'}, 0.9999942779541016], [{'start': 1001, 'end': 1005, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999971389770508], [{'start': 1025, 'end': 1030, 'id': 'CUI-less'}, 0.9999972581863403], [{'start': 1144, 'end': 1148, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999971985816956], [{'start': 1200, 'end': 1204, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999971389770508], [{'start': 1319, 'end': 1323, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.9999969005584717], [{'start': 1422, 'end': 1425, 'id': 'MIM:600480\\tHGNC:11623\\tEnsembl:ENSG00000140262\\tBERN:324936702'}, 0.9999973773956299], [{'start': 1430, 'end': 1433, 'id': 'MIM:147141\\tHGNC:11633\\tEnsembl:ENSG00000071564\\tBERN:324924802'}, 0.9999973773956299], [{'start': 1452, 'end': 1456, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999961853027344], [{'start': 1457, 'end': 1460, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999973773956299], [{'start': 1488, 'end': 1492, 'id': 'MIM:602272\\tHGNC:11634\\tEnsembl:ENSG00000196628\\tBERN:324924402'}, 0.999997079372406], [{'start': 1521, 'end': 1526, 'id': 'CUI-less'}, 0.9999972581863403], [{'start': 1589, 'end': 1599, 'id': 'CUI-less'}, 0.9999966025352478], [{'start': 1631, 'end': 1636, 'id': 'CUI-less'}, 0.9999972581863403]], 'drug': [[{'start': 1218, 'end': 1222, 'id': 'MESH:C026804\\tBERN:317251403'}, 0.999988317489624]], 'species': [[{'start': 1116, 'end': 1124}, 0.9999998807907104], [{'start': 1280, 'end': 1285, 'id': 'NCBI:txid10090'}, 0.9999997615814209], [{'start': 1295, 'end': 1300}, 0.9999998807907104]]}}}\n",
      "{'Status': 'Public on Jun 23, 2015', 'Title': 'The oncofusion protein FUS-ERG targets key hematopoietic regulators and modulates the all-trans retinoic acid signaling pathway in t(16;21) acute myeloid leukemia', 'Organism': 'Homo sapiens', 'Experiment type': 'Genome binding/occupancy profiling by high throughput sequencingExpression profiling by high throughput sequencing', 'Summary': \"Fusion proteins involving the ETS factor ERG have been associated with multiple cancers such as Ewing's sarcoma and prostate cancer. In acute myeloid leukemias harboring  t(16;21) another ERG fusion protein is expressed, FUS-ERG. Here, we found that this FUS-ERG oncofusion protein acts in the context of a heptad of proteins (ERG, FLI1, GATA2, LYL1, LNMO2, RUNX1 and TAL1) central to proper expression of genes involved in maintaining a stem cell hematopoietic phenotype. Moreover, in t(16;21) FUS-ERG co-occupies genomic regions bound by the nuclear receptor heterodimer RXR-RARA inhibiting target gene expression and interfering with hematopoietic differentiation. All-Trans Retinoic Acid treatment of t(16;21) cells as well as FUS-ERG knock down alleviate the myeloid differentiation block. Together, the results suggest that FUS-ERG acts as a transcriptional repressor of the retinoic acid signaling pathway.\", 'Overall design': 'Cell lines were used for RNA extraction for RNA-seq and ChIP experiments for ChIP-seq.', 'Contributor(s)': 'Sotoca AM, Prange K, Reijnders B, Mandoli A, Nguyen LN, Stunnenberg HG, Martens JH', 'Citation(s)': '26148230', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': 'bba673513f3533b7c20a5380c3cf9c2890019c8a4c0b9bd7523bad72-Thread-14989914', 'text': \"Fusion proteins involving the ETS factor ERG have been associated with multiple cancers such as Ewing's sarcoma and prostate cancer. In acute myeloid leukemias harboring  t(16;21) another ERG fusion protein is expressed, FUS-ERG. Here, we found that this FUS-ERG oncofusion protein acts in the context of a heptad of proteins (ERG, FLI1, GATA2, LYL1, LNMO2, RUNX1 and TAL1) central to proper expression of genes involved in maintaining a stem cell hematopoietic phenotype. Moreover, in t(16;21) FUS-ERG co-occupies genomic regions bound by the nuclear receptor heterodimer RXR-RARA inhibiting target gene expression and interfering with hematopoietic differentiation. All-Trans Retinoic Acid treatment of t(16;21) cells as well as FUS-ERG knock down alleviate the myeloid differentiation block. Together, the results suggest that FUS-ERG acts as a transcriptional repressor of the retinoic acid signaling pathway.\", 'denotations': [{'id': ['CUI-less'], 'span': {'begin': 30, 'end': 40}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 41, 'end': 44}, 'obj': 'gene'}, {'id': ['MESH:D002277', 'BERN:255268301'], 'span': {'begin': 80, 'end': 87}, 'obj': 'disease'}, {'id': ['MESH:D012512', 'OMIM:612219', 'BERN:107392101'], 'span': {'begin': 96, 'end': 111}, 'obj': 'disease'}, {'id': ['OMIM:176807', 'MESH:D011471', 'BERN:107535701'], 'span': {'begin': 116, 'end': 131}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 136, 'end': 159}, 'obj': 'disease'}, {'id': ['CUI-less'], 'span': {'begin': 188, 'end': 206}, 'obj': 'gene'}, {'id': ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], 'span': {'begin': 221, 'end': 224}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 225, 'end': 228}, 'obj': 'gene'}, {'id': ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], 'span': {'begin': 255, 'end': 258}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 259, 'end': 262}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 327, 'end': 330}, 'obj': 'gene'}, {'id': ['MIM:193067', 'HGNC:3749', 'Ensembl:ENSG00000151702', 'BERN:323175602'], 'span': {'begin': 332, 'end': 336}, 'obj': 'gene'}, {'id': ['MIM:137295', 'HGNC:4171', 'Ensembl:ENSG00000179348', 'BERN:323327402'], 'span': {'begin': 338, 'end': 343}, 'obj': 'gene'}, {'id': ['MIM:151440', 'HGNC:6734', 'Ensembl:ENSG00000104903', 'BERN:323999702'], 'span': {'begin': 345, 'end': 349}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 351, 'end': 356}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 358, 'end': 363}, 'obj': 'gene'}, {'id': ['MIM:187040', 'HGNC:11556', 'Ensembl:ENSG00000162367', 'BERN:324912902'], 'span': {'begin': 368, 'end': 372}, 'obj': 'gene'}, {'id': ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], 'span': {'begin': 495, 'end': 498}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 499, 'end': 502}, 'obj': 'gene'}, {'id': ['MIM:180245', 'HGNC:10477', 'Ensembl:ENSG00000186350', 'BERN:324635202'], 'span': {'begin': 573, 'end': 576}, 'obj': 'gene'}, {'id': ['MIM:180240', 'HGNC:9864', 'Ensembl:ENSG00000131759', 'BERN:324583702'], 'span': {'begin': 577, 'end': 581}, 'obj': 'gene'}, {'id': ['MESH:D014212', 'BERN:4116003'], 'span': {'begin': 668, 'end': 691}, 'obj': 'drug'}, {'id': ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], 'span': {'begin': 731, 'end': 734}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 735, 'end': 738}, 'obj': 'gene'}, {'id': ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], 'span': {'begin': 830, 'end': 833}, 'obj': 'gene'}, {'id': ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], 'span': {'begin': 834, 'end': 837}, 'obj': 'gene'}, {'id': ['CHEBI:26536', 'BERN:301815603'], 'span': {'begin': 881, 'end': 894}, 'obj': 'drug'}], 'timestamp': 'Sun Jul 18 17:35:07 +0000 2021', 'logits': {'disease': [[{'start': 80, 'end': 87, 'id': 'MESH:D002277\\tBERN:255268301'}, 0.9999996423721313], [{'start': 96, 'end': 111, 'id': 'MESH:D012512\\tOMIM:612219\\tBERN:107392101'}, 1], [{'start': 116, 'end': 131, 'id': 'OMIM:176807\\tMESH:D011471\\tBERN:107535701'}, 0.9999999403953552], [{'start': 136, 'end': 159, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999999403953552]], 'gene': [[{'start': 30, 'end': 40, 'id': 'CUI-less'}, 0.9999698400497437], [{'start': 41, 'end': 44, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999920129776001], [{'start': 188, 'end': 206, 'id': 'CUI-less'}, 0.9999899864196777], [{'start': 221, 'end': 224, 'id': 'MIM:137070\\tHGNC:4010\\tEnsembl:ENSG00000089280\\tBERN:323255002'}, 0.9999971389770508], [{'start': 225, 'end': 228, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999972581863403], [{'start': 255, 'end': 258, 'id': 'MIM:137070\\tHGNC:4010\\tEnsembl:ENSG00000089280\\tBERN:323255002'}, 0.9999971389770508], [{'start': 259, 'end': 262, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999972581863403], [{'start': 327, 'end': 330, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999971389770508], [{'start': 332, 'end': 336, 'id': 'MIM:193067\\tHGNC:3749\\tEnsembl:ENSG00000151702\\tBERN:323175602'}, 0.9999972581863403], [{'start': 338, 'end': 343, 'id': 'MIM:137295\\tHGNC:4171\\tEnsembl:ENSG00000179348\\tBERN:323327402'}, 0.9999973773956299], [{'start': 345, 'end': 349, 'id': 'MIM:151440\\tHGNC:6734\\tEnsembl:ENSG00000104903\\tBERN:323999702'}, 0.9999972581863403], [{'start': 351, 'end': 356, 'id': 'CUI-less'}, 0.9999971389770508], [{'start': 358, 'end': 363, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999973773956299], [{'start': 368, 'end': 372, 'id': 'MIM:187040\\tHGNC:11556\\tEnsembl:ENSG00000162367\\tBERN:324912902'}, 0.9999973773956299], [{'start': 495, 'end': 498, 'id': 'MIM:137070\\tHGNC:4010\\tEnsembl:ENSG00000089280\\tBERN:323255002'}, 0.9999973773956299], [{'start': 499, 'end': 502, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999972581863403], [{'start': 573, 'end': 576, 'id': 'MIM:180245\\tHGNC:10477\\tEnsembl:ENSG00000186350\\tBERN:324635202'}, 0.9999972581863403], [{'start': 577, 'end': 581, 'id': 'MIM:180240\\tHGNC:9864\\tEnsembl:ENSG00000131759\\tBERN:324583702'}, 0.9999972581863403], [{'start': 731, 'end': 734, 'id': 'MIM:137070\\tHGNC:4010\\tEnsembl:ENSG00000089280\\tBERN:323255002'}, 0.9999972581863403], [{'start': 735, 'end': 738, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999971389770508], [{'start': 830, 'end': 833, 'id': 'MIM:137070\\tHGNC:4010\\tEnsembl:ENSG00000089280\\tBERN:323255002'}, 0.9999972581863403], [{'start': 834, 'end': 837, 'id': 'MIM:165080\\tHGNC:3446\\tEnsembl:ENSG00000157554\\tBERN:323100502'}, 0.9999971389770508]], 'drug': [[{'start': 668, 'end': 691, 'id': 'MESH:D014212\\tBERN:4116003'}, 0.9999982714653015], [{'start': 881, 'end': 894, 'id': 'CHEBI:26536\\tBERN:301815603'}, 0.9999942779541016]], 'species': []}}}\n",
      "{'Status': 'Public on Dec 26, 2013', 'Title': 'Analysis of gene expression changes in RN2 acute myeloid leukemia (AML) cells at 24hrs and 48hrs after Ddx5 knockdown.', 'Organism': 'Mus musculus', 'Experiment type': 'Expression profiling by high throughput sequencing', 'Summary': 'Ddx5 inhibition in RN2 cells slows cell proliferation and induces apoptosis within 48-72hrs. The aim of this analysis was to gain insight into how Ddx5 inhibition causes this outcome by analyzing gene expression changes in RN2 cells that occur at early timepoints after Ddx5 inhibition that precedes the timepoint when RN2 proliferation/cell death becomes evident in tissue culture (72hrs after inhibition).', 'Overall design': 'Derivatives of RN2 AML cells were prepared that encode doxycycline-inducible expression of either of two different shRNAs targeting Ddx5 (shDdx5.1322 and shDdx5.2086) or a negative control shRNA that targets Renilla Luciferase (shRen.713).Six independent cultures of each derivative RN2 cell line (shDdx5.1322, shDdx5.2086, or shRen.713) were treated with doxycycline at timepoint 0 days to induce expression of the indicated shRNA in the RN2 cells. Each shRNA is co-expressed with dsRed in a doxycycline-induced manner and flow cytometry analysis indicated that doxycycline induced expression of dsRed and the shRNA in 70-to-80% of the cells in each culture. RNA was isolated from three independent cultures of each derivative RN2 cell line at either 24hrs and 48hrs after dpxycycline treatment. Therefore this study consists of 18 samples, sequencing results from biological triplicate samples of RN2-shDdx5.1322, RN2-shDdx5.2086, and RN2-shRen.713 at 24hrs post-doxycycline and sequencing results from biological triplicate samples of RN2-shDdx5.1322, RN2-shDdx5.2086, and shRen.713 at 48hrs post-doxycycline.', 'Contributor(s)': 'Mazurek A', 'Citation(s)': '24910429', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': '68471c5e3a4c34bcc57403b412d01af1e33f6f43c405e42ffa1119f6-Thread-14989929', 'text': 'Ddx5 inhibition in RN2 cells slows cell proliferation and induces apoptosis within 48-72hrs. The aim of this analysis was to gain insight into how Ddx5 inhibition causes this outcome by analyzing gene expression changes in RN2 cells that occur at early timepoints after Ddx5 inhibition that precedes the timepoint when RN2 proliferation/cell death becomes evident in tissue culture (72hrs after inhibition).', 'denotations': [{'id': ['MIM:180630', 'HGNC:2746', 'Ensembl:ENSG00000108654', 'BERN:323012902'], 'span': {'begin': 0, 'end': 4}, 'obj': 'gene'}, {'id': ['MIM:180630', 'HGNC:2746', 'Ensembl:ENSG00000108654', 'BERN:323012902'], 'span': {'begin': 147, 'end': 151}, 'obj': 'gene'}, {'id': ['MIM:180630', 'HGNC:2746', 'Ensembl:ENSG00000108654', 'BERN:323012902'], 'span': {'begin': 270, 'end': 274}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 319, 'end': 322}, 'obj': 'gene'}], 'timestamp': 'Sun Jul 18 17:35:10 +0000 2021', 'logits': {'disease': [], 'gene': [[{'start': 0, 'end': 4, 'id': 'MIM:180630\\tHGNC:2746\\tEnsembl:ENSG00000108654\\tBERN:323012902'}, 0.999996542930603], [{'start': 147, 'end': 151, 'id': 'MIM:180630\\tHGNC:2746\\tEnsembl:ENSG00000108654\\tBERN:323012902'}, 0.9999970197677612], [{'start': 270, 'end': 274, 'id': 'MIM:180630\\tHGNC:2746\\tEnsembl:ENSG00000108654\\tBERN:323012902'}, 0.9999939203262329], [{'start': 319, 'end': 322, 'id': 'CUI-less'}, 0.886603593826294]], 'drug': [], 'species': []}}}\n",
      "{'Status': 'Public on Jun 20, 2017', 'Title': 'Nascent RNA-seq in Kasumi-1 cells', 'Organism': 'Homo sapiens', 'Experiment type': 'Expression profiling by high throughput sequencing', 'Summary': 'Amino Enhancer of Split (AES) is essential for AML1-ETO induced self-renewal and leukemogenesis. To study the effect of AES on transcription regulation in AML1-ETO expressing Kasumi-1 cells, nascent transcripts in control (shctr) and AES knockdown (shAES) Kasumi-1 cells were labelled with uridine analogue 4-thioduridine with subsequent nascent RNA purification and next generation sequencing (Nascent RNA-seq).', 'Overall design': 'Examination of nascent transcription in Kasumi-1 cells under two conditions with duplicate.', 'Contributor(s)': 'Zhou F, Rohde C, Klein H, MÃ¼ller-Tidow C', 'Citation(s)': '28650479', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': 'b755cdc236f83342073f33238fce77efc94c01a1f9cd45c4705dff34-Thread-14989941', 'text': 'Amino Enhancer of Split (AES) is essential for AML1-ETO induced self-renewal and leukemogenesis. To study the effect of AES on transcription regulation in AML1-ETO expressing Kasumi-1 cells, nascent transcripts in control (shctr) and AES knockdown (shAES) Kasumi-1 cells were labelled with uridine analogue 4-thioduridine with subsequent nascent RNA purification and next generation sequencing (Nascent RNA-seq).', 'denotations': [{'id': ['CHEBI:46882', 'BERN:5540103'], 'span': {'begin': 0, 'end': 5}, 'obj': 'drug'}, {'id': ['CUI-less'], 'span': {'begin': 0, 'end': 23}, 'obj': 'gene'}, {'id': ['MIM:600188', 'HGNC:307', 'Ensembl:ENSG00000104964', 'BERN:323014102'], 'span': {'begin': 25, 'end': 28}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 47, 'end': 51}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 52, 'end': 55}, 'obj': 'gene'}, {'id': ['MIM:600188', 'HGNC:307', 'Ensembl:ENSG00000104964', 'BERN:323014102'], 'span': {'begin': 120, 'end': 123}, 'obj': 'gene'}, {'id': ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], 'span': {'begin': 155, 'end': 159}, 'obj': 'gene'}, {'id': ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], 'span': {'begin': 160, 'end': 163}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 223, 'end': 228}, 'obj': 'gene'}, {'id': ['MIM:600188', 'HGNC:307', 'Ensembl:ENSG00000104964', 'BERN:323014102'], 'span': {'begin': 234, 'end': 237}, 'obj': 'gene'}, {'id': ['CHEBI:16704', 'BERN:4226903'], 'span': {'begin': 290, 'end': 297}, 'obj': 'drug'}, {'id': ['CUI-less'], 'span': {'begin': 307, 'end': 321}, 'obj': 'drug'}], 'timestamp': 'Sun Jul 18 17:35:13 +0000 2021', 'logits': {'disease': [], 'gene': [[{'start': 0, 'end': 23, 'id': 'CUI-less'}, 0.9999968409538269], [{'start': 25, 'end': 28, 'id': 'MIM:600188\\tHGNC:307\\tEnsembl:ENSG00000104964\\tBERN:323014102'}, 0.9999971389770508], [{'start': 47, 'end': 51, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999967813491821], [{'start': 52, 'end': 55, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999972581863403], [{'start': 120, 'end': 123, 'id': 'MIM:600188\\tHGNC:307\\tEnsembl:ENSG00000104964\\tBERN:323014102'}, 0.6840946078300476], [{'start': 155, 'end': 159, 'id': 'MIM:151385\\tHGNC:10471\\tEnsembl:ENSG00000159216\\tBERN:325374902'}, 0.9999953508377075], [{'start': 160, 'end': 163, 'id': 'MIM:133435\\tHGNC:1535\\tEnsembl:ENSG00000079102\\tBERN:325375702'}, 0.9999964237213135], [{'start': 223, 'end': 228, 'id': 'CUI-less'}, 0.9999940395355225], [{'start': 234, 'end': 237, 'id': 'MIM:600188\\tHGNC:307\\tEnsembl:ENSG00000104964\\tBERN:323014102'}, 0.9973040819168091]], 'drug': [[{'start': 0, 'end': 5, 'id': 'CHEBI:46882\\tBERN:5540103'}, 0.999995231628418], [{'start': 290, 'end': 297, 'id': 'CHEBI:16704\\tBERN:4226903'}, 0.9997548460960388], [{'start': 307, 'end': 321, 'id': 'CUI-less'}, 0.9999982714653015]], 'species': []}}}\n",
      "{'Status': 'Public on Mar 12, 2018', 'Title': 'LKB1, Salt-Inducible Kinases, and MEF2C are linked dependencies in acute myeloid leukemia', 'Organism': 'Homo sapiens', 'Experiment type': 'Expression profiling by high throughput sequencingGenome binding/occupancy profiling by high throughput sequencing', 'Summary': 'This SuperSeries is composed of the SubSeries listed below.', 'Overall design': 'Refer to individual Series', 'Citation(s)': '29526696', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': '36cc384bc12c9955739a653985faa40a44baee452ab6e8e5f40f1b33-Thread-14989954', 'text': 'This SuperSeries is composed of the SubSeries listed below.', 'denotations': [], 'timestamp': 'Sun Jul 18 17:35:14 +0000 2021', 'logits': {'disease': [], 'gene': [], 'drug': [], 'species': []}}}\n",
      "{'Status': 'Public on Feb 27, 2012', 'Title': 'GSK-3A and GSK-3B knockdown in AML cell lines', 'Organism': 'Homo sapiens', 'Experiment type': 'Expression profiling by array', 'Summary': 'Gene expression data from AML cell lines, MOLM-14, U937, THP-1 and HL-60, that were infected with a scrambled control hairpin (shControl), two shRNAs directed against GSK-3B (shGSK3B_1 and shGSK3B_2), or two shRNAs directed against GSK-3A (shGSK3A_5 and shGSK3A_6).Acute myeloid leukemia (AML) is the most common form of acute leukemia in adults. Long-term survival of patients with AML has changed little over the past decade, necessitating the identification and validation of new AML targets. Integration of genomic approaches with small-molecule and genetic-based high-throughput screening holds the promise of improved discovery of candidate targets for cancer therapy. Here, we identified a role for glycogen synthase kinase 3A (GSK-3A) in AML by performing two independent small-molecule library screens and an shRNA screen for perturbations that induced a differentiation expression signature in AML cells. GSK-3 is a serine-threonine kinase involved in diverse cellular processes including differentiation, signal transduction, cell cycle regulation, and proliferation. We demonstrated that specific loss of GSK-3A induced differentiation in AML by multiple measurements, including induction of gene expression signatures, morphological changes, and cell surface markers consistent with myeloid maturation. GSK-3Aâ\\x80\\x93specific suppression also led to impaired growth and proliferation in vitro, induction of apoptosis, loss of colony formation in methylcellulose, and anti-AML activity in vivo. Although the role of GSK-3B has been well studied in cancer development, these studies support a role for GSK-3A in AML.', 'Overall design': 'The AML cell lines, MOLM-14, U937, THP-1 and HL-60, were infected with a scrambled control hairpin (shControl), two shRNAs directed against GSK-3B (shGSK3B_1 and shGSK3B_2), and two shRNAs directed against GSK-3A (shGSK3A_5 and shGSK3A_6).', 'Contributor(s)': 'Stegmaier K, Banerji V, Frumm SM, Ross KN', 'Citation(s)': '22326953', 'annotate': {'project': 'BERN', 'sourcedb': '', 'sourceid': '8eb887ab2f93e8340bd25bed541dbb058167464b20234045caf43347-Thread-14989959', 'text': 'Gene expression data from AML cell lines, MOLM-14, U937, THP-1 and HL-60, that were infected with a scrambled control hairpin (shControl), two shRNAs directed against GSK-3B (shGSK3B_1 and shGSK3B_2), or two shRNAs directed against GSK-3A (shGSK3A_5 and shGSK3A_6).Acute myeloid leukemia (AML) is the most common form of acute leukemia in adults. Long-term survival of patients with AML has changed little over the past decade, necessitating the identification and validation of new AML targets. Integration of genomic approaches with small-molecule and genetic-based high-throughput screening holds the promise of improved discovery of candidate targets for cancer therapy. Here, we identified a role for glycogen synthase kinase 3A (GSK-3A) in AML by performing two independent small-molecule library screens and an shRNA screen for perturbations that induced a differentiation expression signature in AML cells. GSK-3 is a serine-threonine kinase involved in diverse cellular processes including differentiation, signal transduction, cell cycle regulation, and proliferation. We demonstrated that specific loss of GSK-3A induced differentiation in AML by multiple measurements, including induction of gene expression signatures, morphological changes, and cell surface markers consistent with myeloid maturation. GSK-3A   specific suppression also led to impaired growth and proliferation in vitro, induction of apoptosis, loss of colony formation in methylcellulose, and anti-AML activity in vivo. Although the role of GSK-3B has been well studied in cancer development, these studies support a role for GSK-3A in AML.', 'denotations': [{'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 26, 'end': 29}, 'obj': 'disease'}, {'id': ['MIM:605004', 'HGNC:4617', 'Ensembl:ENSG00000082701', 'BERN:323566402'], 'span': {'begin': 167, 'end': 173}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 175, 'end': 184}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 189, 'end': 198}, 'obj': 'gene'}, {'id': ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], 'span': {'begin': 232, 'end': 238}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 240, 'end': 249}, 'obj': 'gene'}, {'id': ['CUI-less'], 'span': {'begin': 254, 'end': 263}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 265, 'end': 287}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 289, 'end': 292}, 'obj': 'disease'}, {'id': ['BERN:256418201'], 'span': {'begin': 321, 'end': 335}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 383, 'end': 386}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 483, 'end': 486}, 'obj': 'disease'}, {'id': ['MESH:D002277', 'BERN:255268301'], 'span': {'begin': 659, 'end': 665}, 'obj': 'disease'}, {'id': ['CUI-less'], 'span': {'begin': 706, 'end': 733}, 'obj': 'gene'}, {'id': ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], 'span': {'begin': 735, 'end': 741}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 746, 'end': 749}, 'obj': 'disease'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 904, 'end': 907}, 'obj': 'disease'}, {'id': ['CUI-less'], 'span': {'begin': 915, 'end': 920}, 'obj': 'gene'}, {'id': ['CHEBI:17115', 'BERN:4176303'], 'span': {'begin': 926, 'end': 932}, 'obj': 'drug'}, {'id': ['CUI-less'], 'span': {'begin': 926, 'end': 949}, 'obj': 'gene'}, {'id': ['CHEBI:26986', 'BERN:4184203'], 'span': {'begin': 933, 'end': 942}, 'obj': 'drug'}, {'id': ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], 'span': {'begin': 1117, 'end': 1123}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 1151, 'end': 1154}, 'obj': 'disease'}, {'id': ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], 'span': {'begin': 1316, 'end': 1322}, 'obj': 'gene'}, {'id': ['MIM:605004', 'HGNC:4617', 'Ensembl:ENSG00000082701', 'BERN:323566402'], 'span': {'begin': 1523, 'end': 1529}, 'obj': 'gene'}, {'id': ['MESH:D002277', 'BERN:255268301'], 'span': {'begin': 1555, 'end': 1561}, 'obj': 'disease'}, {'id': ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], 'span': {'begin': 1608, 'end': 1614}, 'obj': 'gene'}, {'id': ['MESH:D015470', 'OMIM:601626', 'BERN:107418901'], 'span': {'begin': 1618, 'end': 1621}, 'obj': 'disease'}], 'timestamp': 'Sun Jul 18 17:35:19 +0000 2021', 'logits': {'disease': [[{'start': 26, 'end': 29, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 265, 'end': 287, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998211860657], [{'start': 289, 'end': 292, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 321, 'end': 335, 'id': 'BERN:256418201'}, 0.9999998807907104], [{'start': 383, 'end': 386, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 483, 'end': 486, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 659, 'end': 665, 'id': 'MESH:D002277\\tBERN:255268301'}, 0.9999998807907104], [{'start': 746, 'end': 749, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 904, 'end': 907, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999996423721313], [{'start': 1151, 'end': 1154, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104], [{'start': 1555, 'end': 1561, 'id': 'MESH:D002277\\tBERN:255268301'}, 0.9999998807907104], [{'start': 1618, 'end': 1621, 'id': 'MESH:D015470\\tOMIM:601626\\tBERN:107418901'}, 0.9999998807907104]], 'gene': [[{'start': 167, 'end': 173, 'id': 'MIM:605004\\tHGNC:4617\\tEnsembl:ENSG00000082701\\tBERN:323566402'}, 0.9999971389770508], [{'start': 175, 'end': 184, 'id': 'CUI-less'}, 0.9999969601631165], [{'start': 189, 'end': 198, 'id': 'CUI-less'}, 0.9999969005584717], [{'start': 232, 'end': 238, 'id': 'MIM:606784\\tHGNC:4616\\tEnsembl:ENSG00000105723\\tBERN:323566302'}, 0.9999971389770508], [{'start': 240, 'end': 249, 'id': 'CUI-less'}, 0.9999969601631165], [{'start': 254, 'end': 263, 'id': 'CUI-less'}, 0.999997079372406], [{'start': 706, 'end': 733, 'id': 'CUI-less'}, 0.9999969601631165], [{'start': 735, 'end': 741, 'id': 'MIM:606784\\tHGNC:4616\\tEnsembl:ENSG00000105723\\tBERN:323566302'}, 0.9999969601631165], [{'start': 915, 'end': 920, 'id': 'CUI-less'}, 0.9999971389770508], [{'start': 926, 'end': 949, 'id': 'CUI-less'}, 0.999997079372406], [{'start': 1117, 'end': 1123, 'id': 'MIM:606784\\tHGNC:4616\\tEnsembl:ENSG00000105723\\tBERN:323566302'}, 0.9999969601631165], [{'start': 1316, 'end': 1322, 'id': 'MIM:606784\\tHGNC:4616\\tEnsembl:ENSG00000105723\\tBERN:323566302'}, 0.9999971389770508], [{'start': 1523, 'end': 1529, 'id': 'MIM:605004\\tHGNC:4617\\tEnsembl:ENSG00000082701\\tBERN:323566402'}, 0.9999969601631165], [{'start': 1608, 'end': 1614, 'id': 'MIM:606784\\tHGNC:4616\\tEnsembl:ENSG00000105723\\tBERN:323566302'}, 0.9999969601631165]], 'drug': [[{'start': 926, 'end': 932, 'id': 'CHEBI:17115\\tBERN:4176303'}, 0.9999955892562866], [{'start': 933, 'end': 942, 'id': 'CHEBI:26986\\tBERN:4184203'}, 0.9999959468841553]], 'species': [[{'start': 369, 'end': 377}, 0.9999998807907104]]}}}\n"
     ]
    }
   ],
   "source": [
    "for i in data:\n",
    "    print(i)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Creating and appending it to the Database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Connected successfully!!!\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5A48>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E57C8>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5108>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5708>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5588>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5A48>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5688>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5248>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E57C8>\n",
      "Data inserted with record ids <pymongo.results.InsertOneResult object at 0x0000017F3B6E5A88>\n"
     ]
    }
   ],
   "source": [
    "from pymongo import MongoClient\n",
    "  \n",
    "try:\n",
    "    conn = MongoClient()\n",
    "    print(\"Connected successfully!!!\")\n",
    "except:  \n",
    "    print(\"Could not connect to MongoDB\")\n",
    "  \n",
    "# database\n",
    "db = conn.database\n",
    "  \n",
    "# Created or Switched to collection names: my_gfg_collection\n",
    "collection = db.databsae\n",
    "for i in data:\n",
    "    rec_id=collection.insert_one(i)\n",
    "    print(\"Data inserted with record ids\",rec_id)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# SEARCH QUERY"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "id's are :\n",
      "[['CUI-less'], ['MIM:164790', 'HGNC:7989', 'Ensembl:ENSG00000213281', 'BERN:324183902'], ['MIM:159555', 'HGNC:7132', 'Ensembl:ENSG00000118058', 'BERN:324047002'], ['CUI-less'], ['CUI-less'], ['MIM:159555', 'HGNC:7132', 'Ensembl:ENSG00000118058', 'BERN:324047002'], ['MIM:159558', 'HGNC:7136', 'Ensembl:ENSG00000171843', 'BERN:324047502'], ['MIM:189990', 'HGNC:7545', 'Ensembl:ENSG00000118513', 'BERN:324141502'], ['BERN:326404802'], ['CUI-less'], ['MIM:601231', 'HGNC:3942', 'Ensembl:ENSG00000198793', 'BERN:323251002'], ['MIM:603014', 'HGNC:6847', 'Ensembl:ENSG00000076984', 'BERN:324467602'], ['CUI-less'], ['CUI-less'], ['CUI-less'], ['CUI-less'], ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], ['MIM:600160', 'HGNC:1787', 'Ensembl:ENSG00000147889', 'BERN:321173202'], ['MIM:600160', 'HGNC:1787', 'Ensembl:ENSG00000147889', 'BERN:321173202'], ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], ['CUI-less'], ['CUI-less'], ['MIM:604420', 'HGNC:4901', 'Ensembl:ENSG00000152804', 'BERN:323596902'], ['MIM:600618', 'HGNC:3495', 'Ensembl:ENSG00000139083', 'BERN:323103602'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:600618', 'HGNC:3495', 'Ensembl:ENSG00000139083', 'BERN:323103602'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:600618', 'HGNC:3495', 'Ensembl:ENSG00000139083', 'BERN:323103602'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['CUI-less'], ['CUI-less'], ['MIM:600480', 'HGNC:11623', 'Ensembl:ENSG00000140262', 'BERN:324936702'], ['MIM:147141', 'HGNC:11633', 'Ensembl:ENSG00000071564', 'BERN:324924802'], ['CUI-less'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['CUI-less'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['CUI-less'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['CUI-less'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['MIM:600480', 'HGNC:11623', 'Ensembl:ENSG00000140262', 'BERN:324936702'], ['MIM:147141', 'HGNC:11633', 'Ensembl:ENSG00000071564', 'BERN:324924802'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['MIM:602272', 'HGNC:11634', 'Ensembl:ENSG00000196628', 'BERN:324924402'], ['CUI-less'], ['CUI-less'], ['CUI-less'], ['CUI-less'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['CUI-less'], ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['MIM:193067', 'HGNC:3749', 'Ensembl:ENSG00000151702', 'BERN:323175602'], ['MIM:137295', 'HGNC:4171', 'Ensembl:ENSG00000179348', 'BERN:323327402'], ['MIM:151440', 'HGNC:6734', 'Ensembl:ENSG00000104903', 'BERN:323999702'], ['CUI-less'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:187040', 'HGNC:11556', 'Ensembl:ENSG00000162367', 'BERN:324912902'], ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['MIM:180245', 'HGNC:10477', 'Ensembl:ENSG00000186350', 'BERN:324635202'], ['MIM:180240', 'HGNC:9864', 'Ensembl:ENSG00000131759', 'BERN:324583702'], ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['MIM:137070', 'HGNC:4010', 'Ensembl:ENSG00000089280', 'BERN:323255002'], ['MIM:165080', 'HGNC:3446', 'Ensembl:ENSG00000157554', 'BERN:323100502'], ['MIM:180630', 'HGNC:2746', 'Ensembl:ENSG00000108654', 'BERN:323012902'], ['MIM:180630', 'HGNC:2746', 'Ensembl:ENSG00000108654', 'BERN:323012902'], ['MIM:180630', 'HGNC:2746', 'Ensembl:ENSG00000108654', 'BERN:323012902'], ['CUI-less'], ['CUI-less'], ['MIM:600188', 'HGNC:307', 'Ensembl:ENSG00000104964', 'BERN:323014102'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['MIM:600188', 'HGNC:307', 'Ensembl:ENSG00000104964', 'BERN:323014102'], ['MIM:151385', 'HGNC:10471', 'Ensembl:ENSG00000159216', 'BERN:325374902'], ['MIM:133435', 'HGNC:1535', 'Ensembl:ENSG00000079102', 'BERN:325375702'], ['CUI-less'], ['MIM:600188', 'HGNC:307', 'Ensembl:ENSG00000104964', 'BERN:323014102'], ['MIM:605004', 'HGNC:4617', 'Ensembl:ENSG00000082701', 'BERN:323566402'], ['CUI-less'], ['CUI-less'], ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], ['CUI-less'], ['CUI-less'], ['CUI-less'], ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], ['CUI-less'], ['CUI-less'], ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302'], ['MIM:605004', 'HGNC:4617', 'Ensembl:ENSG00000082701', 'BERN:323566402'], ['MIM:606784', 'HGNC:4616', 'Ensembl:ENSG00000105723', 'BERN:323566302']]\n"
     ]
    }
   ],
   "source": [
    "keyword=\"gene\"\n",
    "ans=[]\n",
    "for i in data:\n",
    "    if 'annotate' in i.keys():\n",
    "        for k in i['annotate']['denotations']:\n",
    "            if k['obj']==keyword:\n",
    "                ans.append(k['id'])\n",
    "        \n",
    "print(\"id's are :\")\n",
    "print(ans)\n",
    "ans=[]\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
